mappings/net/minecraft/client/render/entity/VillagerEntityRenderLayer.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS dlg net/minecraft/client/render/entity/VillagerEntityRenderLayer
+	FIELD a levelToId Lit/unimi/dsi/fastutil/ints/Int2ObjectMap;
```

Comment:
Nuh-uh! That's a static final!

Reply:
I thought a static final field was not necessarily immutable, will change when I get back on enigma https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Our convention to date was to call all static finals UPPER_CASE. If you want to change *convention*, that's probably an issue for another PR